src/json/UnitSettings.schema.json

```
@@ -37,6 +37,33 @@
           },
           "additionalProperties": false,
           "required": ["isCancellable", "isRedeemable", "isExpirable"]
+        },
+        "translatableConfig": {
+          "type": "object",
+          "title": "Translatable Configuration",
+          "description": "Locale-specific configuration for rewards of this unit",
+          "properties": {
+            "prettyValueFormat": {
+              "type": "object",
+              "title": "Pretty Value Format",
+              "description": "Format for prettyValue for rewards of this unit",
+              "properties": {
+                "templateType": {
+                  "title": "Template Type",
+                  "description": "The type of this template, e.g. ICU.",
+                  "enum": ["ICU"]
+                },
+                "template": {
+                  "type": "string",
+                  "title": "Template",
+                  "description": "The template string for reward prettyValue"
+                }
+              },
+              "additionalProperties": false,
+              "required": ["templateType", "template"]
```

Default should be "ICU". I don't think it needs to be specified.

```suggestion
              "required": ["template"]
```